src/api/db/models/Image.js

```diff
@@ -96,6 +96,32 @@ export class ImageModel {
     }
   }
 
+  static async deleteImage(input, context) {
+    try {
+      const s3 = new S3.S3Client({ region: process.env.AWS_DEFAULT_REGION });
+
+      // Ensure Image is part of a project that the user has access to
+      await ImageModel.queryById(input.imageId, context);
+
+      await Promise.all(['medium', 'original', 'small'].map((size) => {
+        return s3.send(new S3.DeleteObjectCommand({
+          Bucket: `animl-images-serving-${process.env.STAGE}`,
+          Key: `${size}/${input.imageId}.jpg`
```

Comment:
Does this work? That doesn't look like it matches the serving bucket Key template (see [image-ingest](https://github.com/tnc-ca-geo/animl-ingest/blob/c75036746ca94b7c503d34cf62cc126eb4d7a9d1/ingest-image/task.js#L267)). We also in theory support .pngs so probably don't want to hard code the file extensions.